ci/jenkins/test.sh

```diff
@@ -839,6 +852,23 @@ function run_conformance {
     kubectl rollout status daemonset/antrea-agent -n kube-system
 
     set +e
+    # Conformance DNS test cases fail in IPv6 Kind cluster because here CoreDNS Pods are IPv6 Pods 
+    # and it can't reach IPv4 upstream nameserver.
+    # Note: Default CoreDNS ConfigMap uses /etc/resolv.conf to get upstream nameservers listed in resolv.conf file on the host.
+    # The upstream nameservers are accessed by IPv4. When single stack IPv6 Pods try to connect IPv4 nameserver, it return servfail.
+    # With the new ConfigMap, it will forward DNS query to "vmware.com:53" for the lab Jenkins and `compute.internal:53` for the cloud Jenkins,
+    # and it will return 'NXDOMAIN' so that client keep searching.
+    # For more detail: https://github.com/kubernetes/kubernetes/issues/94794 
```

Comment:
The explanation doesn't make sense to me. I think `vmware.com:53` and `compute.internal:53` are not valid DNS servers you can forward queries to. They should mean queries with these suffix are handled separately. 
And if the coreDNS pod can't access upstream IPv4 DNS servers anyway, how could the tests eventually pass given they won't get a valid DNS answer?

Reply:
I found the tests don't really need to resolve external domains so they don't valid upstream server anyway. The workaround is just to ensure it can continue searching. I'm thinking, instead of doing the coredns config hack, can we remove the search domain when setting up the cluster instead?

Reply:
Hi @tnqn , Yes we can remove the search domain from resolve.conf file during creation of kind cluster, one way to do like:
we can create a `custom-resolv.conf` file and will declare a `--node-config` flag in setup-kind.sh script to pass `custom-resolv.conf` file

and in the script we can do something like:
`if [[ "$NODE_CONFIG" != "" ]]; then
    echo -e "  extraMounts:\n  - hostPath: $NODE_CONFIG\n  - containerPath: /etc/resolv.conf" >> $config_file 
fi`

other way is that we can create the kind cluster and then copy `custom-resolv.conf` file  on each node using docker cmds..

could you please correct me?

Reply:
> I found the tests don't really need to resolve external domains so they don't valid upstream server anyway. The workaround is just to ensure it can continue searching. I'm thinking, instead of doing the coredns config hack, can we remove the search domain when setting up the cluster instead?

@tnqn , Right, for running the tests we do not really need to resolve external domains , however we need to capture those requests and handle with NXDOMAIN, as we know NXDOMAIN (Non-Existent Domain) is a DNS response code that indicates that the domain name queried does not need to exist and we have same test usecase here where in external dns queries are failing the tests, there is thread for same for similar issue we faced in this PR https://github.com/kubernetes/kubernetes/issues/94794#issuecomment-694899503

Reply:
understood, working on it, thanks!